### DISCUSSION TITLE
I recently updated to 0.5.13 and am getting error message "Module '"llamaindex"' has no exported member 'Document'"

### DISCUSSION THREAD

**Discussion by oakesdisaac** [run-llama/LlamaIndexTS#1105] [General]
I recently updated to 0.5.13 and am getting error message "Module '"llamaindex"' has no exported member 'Document'"
When using vs 0.3.17, I didn't experience this issue. 

I'm using node v22.2.0

I'm using the following dependencies in my package.json

  "dependencies": {
    "@aws-sdk/client-s3": "^3.622.0",
    "@types/textract": "^2.4.5",
    "cors": "^2.8.5",
    "express": "^4.19.2",
    "llamaindex": "^0.5.13",
    "mammoth": "^1.8.0",
    "pdf.js-extract": "^0.2.1",
    "textract": "^2.5.0"
  },
  "devDependencies": {
    "@types/cors": "^2.8.17",
    "@types/express": "^4.17.21",
    "@types/node": "^22.1.0",
    "@types/pdf-parse": "^1.1.4",
    "copyfiles": "^2.4.1",
    "ncp": "^2.0.0",
    "rimraf": "^6.0.1",
    "ts-node-dev": "^2.0.0",
    "typescript": "^5.5.4"
  }
  
Any help is appreciated.

**Comment by marcusschiesser** [comment]
@oakesdisaac it's now in the core package, so you'll need to use `import { Document } from "@llamaindex/core/schema";` 
@himself65 how about also exporting it from `llamaindex`?

**Reply by himself65** [reply]
Yeah, we could

**Reply by himself65** [reply]
https://github.com/run-llama/LlamaIndexTS/pull/1112

**Reply by oakesdisaac** [reply]
Thanks!

**Comment by oakesdisaac** [comment]
I'm unable to to import from "@llamaindex/core/schema" or "@llamaindex"

**Reply by oakesdisaac** [reply]
@himself65

**Reply by oakesdisaac** [reply]
I ran 

npm install @llamaindex/core

and then was able to use

import { Document } from "@llamaindex/core/dist/schema";

but that causes an error message.

**Reply by himself65** [reply]
change your moduleResolution to bundler in tsconfig
